lib/src/rules/avoid_renaming_method_parameters.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r"Don't rename parameters of overrided methods.";
+
+const _details = r'''
+
+**DON'T** rename parameters of overrided methods.
```

Comment:
nit: "overrided" --> "overridden"

Also, it might be useful to provide some rationale. The specific reason for this lint is that methods that override another method, but do not have their own documentation comment, will inherit the overridden method's comment when dartdoc produces documentation. If the inherited method contains the name of the parameter (in square brackets), then dartdoc cannot link it correctly.